diff --git a/dist/services/payment-module.js b/dist/services/payment-module.js
index 380dd8311a40b7bacdc8272e0afd98959274ffb5..e91ec03b8a9e575435c09d3bc098fcb491302ba2 100644
--- a/dist/services/payment-module.js
+++ b/dist/services/payment-module.js
@@ -515,6 +515,7 @@ class PaymentModuleService extends utils_1.ModulesSdkUtils.MedusaService(generat
             amount: refund.raw_amount,
             context: {
                 idempotency_key: refund.id,
+                refund_metadata: refund.metadata,
             },
         });
         await this.paymentService_.update({ id: payment.id, data: paymentData.data }, sharedContext);
